Automattic/woocommerce-payments
# feat: add digital wallets section

Fixes https://github.com/Automattic/woocommerce-payments/issues/1503

#### Changes proposed in this Pull Request









Adds the "Digital wallets" section to the settings.
UI more closely match the newer Figma designs (compared to ticket description).

Waiting for https://github.com/Automattic/woocommerce-payments/pull/1690 to get merged before implementing [these sections](https://d.pr/i/5t7L2o) on the description (since it touches the same files).


#### Testing instructions





- Ensure you have the "grouped settings" flag enabled in WCPay Dev Tools
- Visit http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout
- Section is displayed

![2021-04-27 08 21 12](https://user-images.githubusercontent.com/273592/116249471-79499180-a732-11eb-864a-ce15edb1824a.gif)

-------------------
